src/Select.soy

```diff
@@ -38,6 +40,7 @@
 			{param header kind="html"}
 				<button
 					class="{$buttonClass ? $buttonClass : ''} dropdown-select"
+					disabled="{$disabled}"
```

Comment:
I've tried to follow the behaviour of a real `<select>`. So, only the button is disabled, since, there is a different way to disabled the `<options>`. I am still working on it. Can you merge this PR without it in case everything is fine?